hack/update-bazel-gcs-mirror.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Can we fold this into a `bazel run //:update-workspace-cache` rule?

Given that this is bazel-centric, providing a bazel-centric UX seems appropriate.

Reply:
Would need to be unsandboxed for gsutil to work, but I guess it could have a rule?

Reply:
`bazel run --direct_run` might be closer to what you want?